app/presenters/return-logs/setup/start.presenter.js

```diff
@@ -0,0 +1,84 @@
+'use strict'
+
+/**
+ * Formats the data ready for presenting in the `/return-log-edit/{sessionId}/start` page
+ * @module StartPresenter
+ */
+
+const { formatAbstractionPeriod, formatLongDate } = require('../../base.presenter.js')
+
+/**
+ * Formats the data ready for presenting in the `/return-log-edit/{sessionId}/start` page
+ *
+ * @param {module:SessionModel} session - The session instance to format
+ *
+ * @returns {object} page data needed for the `/return-log-edit/{sessionId}/start` page
+ */
+function go(session) {
+  const {
+    dueDate,
+    endDate,
+    licenceId,
+    licenceRef,
+    periodStartDay,
+    periodStartMonth,
+    periodEndDay,
+    periodEndMonth,
+    purposes,
+    receivedDate,
+    returnLogId,
+    returnReference,
+    siteDescription,
+    startDate,
+    status,
+    twoPartTariff,
+    whatToDo
+  } = session
+
+  return {
+    abstractionPeriod: `From ${formatAbstractionPeriod(
+      periodStartDay,
+      periodStartMonth,
+      periodEndDay,
+      periodEndMonth
+    )}`,
+    displayRecordReceipt: receivedDate === null,
+    licenceId,
+    licenceRef,
+    pageTitle: 'Abstraction return',
+    purposes,
+    returnLogId,
```

Comment:
Dont need to return returnLogId directly (might need to be in the backlink though)

Reply:
I've removed it as it wasn't needed. It was left over from when the page was being built to edit a return log.